MANUAL.txt

```diff
@@ -1813,6 +1813,16 @@ on the output format, and include the following:
 `body`
 :   body of document
 
+`date-meta`
+:   the `date` variable converted to ISO 8601 YYYY-MM-DD,
+    included in all HTML based formats (dzslides, epub,
+    html, html4, html5, revealjs, s5, slideous, slidy).
+    The recognized formats for `date` are: `mm/dd/yyyy`,
+    `mm/dd/yy`, `yyyy-mm-dd` (ISO 8601), `dd MM yyyy`
+    (e.g. either `02 Apr 2018` or `02 April 2018`),
+    `MM dd, yyyy` (e.g. `Apr. 02, 2018` or `April 02, 2018),
+    `yyyy[mm[dd]]]` (e.g. `20180402, `201804` or `2018`)
```

Comment:
Can you insert a period at the end of this line?

Reply:
Done.